<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lekaren</class>
 <widget class="QWidget" name="Lekaren">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>527</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>814</width>
    <height>527</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>814</width>
    <height>527</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Lekaren</string>
  </property>
  <widget class="QGroupBox" name="groupBoxCely">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>791</width>
     <height>511</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QGroupBox" name="groupBoxObjednavka">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>421</width>
      <height>431</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QTableWidget" name="tableWidgetObjednavka">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>401</width>
       <height>411</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>401</width>
       <height>411</height>
      </size>
     </property>
     <property name="gridStyle">
      <enum>Qt::SolidLine</enum>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>50</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>133</number>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>27</number>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>27</number>
     </attribute>
     <column>
      <property name="text">
       <string>Liek</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Cena</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Počet kusov</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxProdukty">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>9</y>
      <width>171</width>
      <height>431</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QListWidget" name="listLieky">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>151</width>
       <height>411</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>151</width>
       <height>411</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxKusy">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>172</width>
      <height>48</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Počet kusov:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxKusy">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::UpDownArrows</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBoxZamestnanec">
    <property name="geometry">
     <rect>
      <x>639</x>
      <y>100</y>
      <width>130</width>
      <height>236</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Objednávky</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignHCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushZobrazit">
       <property name="text">
        <string>Zobraziť</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushFaktura">
       <property name="text">
        <string>Faktúra</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushPotvrdit">
       <property name="text">
        <string>Potvrdiť</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Zmena pozície</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboStaff">
       <item>
        <property name="text">
         <string>Farmaceut</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Laborant</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Sanitar</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushZmenit">
       <property name="text">
        <string>Zmeniť</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBoxKusy_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>141</width>
      <height>83</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QPushButton" name="pushOdhlasenie">
       <property name="text">
        <string>Odhlásiť sa</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Cena:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxCena">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>450</y>
      <width>421</width>
      <height>52</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushObjednat">
       <property name="text">
        <string>Objednat</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushKosik">
       <property name="text">
        <string>Košík</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushOdstranit">
       <property name="text">
        <string>Odstrániť</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushObjednavka">
       <property name="text">
        <string>Objednávka</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>339</y>
      <width>151</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Zmena údajov</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Meno a priezvisko</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineName">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Adresa</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineAdress">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushZmena">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Potvrdiť</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
